data/report_templates/project_summary.md

```diff
@@ -173,10 +173,10 @@ The naming of the files follow the convention:
 [NGI-NAME]_[BCL-CONVERSION-ID]_[LANE]_[READ]_[VOLUME].fastq.gz
 ```
 
-* _NGI-NAME:_ Internal NGI sample indentifier
-* _BCL-CONVERSION-ID:_ Indentifier set by bcl2fastq tool while demultiplexing
+* _NGI-NAME:_ Internal NGI sample identifier
+* _BCL-CONVERSION-ID:_ identifier set by bcl2fastq tool while demultiplexing
```

Comment:
Capitalise I in identifier like in the other lines.

Reply:
Ooooops that's what happens when you use find and replace 😊